src/components/Article/index.tsx

```diff
@@ -26,10 +25,16 @@ interface Props {
   children?: React.ReactNode;
 }
 
-const NAV_HEIGHT = 72;
-
 const mdxComponents = { pre: Codebox, inlineCode: InlineCode, a: MdxLink };
 
+const renderBlogAuthors = (date?: string, authors?: BlogPostAuthor[]) => (
```

Comment:
You can flip these and make authors required and then you wont need `authors ?? []`

Reply:
But `authors` is allowed to be `undefined` in the query, but the component requires it be defined 🤔

Reply:
it shouldn't be undefined both /learn pages and any blog page should always have an author

Reply:
What is this username? Is that you @benhalverson?

Afaik, in theory, yes, but still the frontmatter field could be empty. But I'll check.

Reply:
😂 ya I was on my work mac last night.